packages/runtime/lib/schema.js

```diff
@@ -87,6 +91,11 @@ const platformaticRuntimeSchema = {
       }
     },
     services,
+    workers: {
+      type: 'number',
+      minimum: 1,
+      default: 1
+    },
```

Comment:
What does this configure? The default number of workers for each autoloaded service?

Reply:
Kinda: the default for any service (autoloaded or not) which does not define the value explicitly.